tools/pepsi/src/aliveness.rs

```diff
@@ -82,10 +83,14 @@ fn print_summary(states: &AlivenessList) {
                 } else {
                     DISCHARGING_ICON
                 };
-
                 output.push_str(&format!("{icon} {charge}%{:SPACING$}", ""))
             }
-        }
+        } else {
+            output.push_str(&format!(
+                "{DISCHARGING_ICON}{UNKNOWN_CHARGE}{:SPACING$}",
```

Comment:
Consider merging `{DISCHARGING_ICON}{UNKNOWN_CHARGE}` into a single constant with the same content, e.g. `{UNKNOWN_CHARGE_ICON}`. This makes changing this one independent of changing the discharging icon.

Reply:
I merged them together.